examples/bri-3/prisma/migrations/20240124083845_add_public_key_to_bpi_subject/migration.sql

```diff
@@ -0,0 +1,27 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `publicKey` on the `BpiSubject` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "BpiSubject" DROP COLUMN "publicKey";
+
+-- CreateTable
+CREATE TABLE "PublicKey" (
+    "id" TEXT NOT NULL,
+    "type" TEXT NOT NULL,
```

Comment:
what is type here? like `EdDSAed25519` or what would that look like such that the right algorithm and curve can be identified? @biscuitdey.